src/components/magicleap-controls.js

```diff
@@ -0,0 +1,192 @@
+var bind = require('../utils/bind');
+var registerComponent = require('../core/component').registerComponent;
+
+var trackedControlsUtils = require('../utils/tracked-controls');
+var checkControllerPresentAndSetup = trackedControlsUtils.checkControllerPresentAndSetup;
+var emitIfAxesChanged = trackedControlsUtils.emitIfAxesChanged;
+var onButtonEvent = trackedControlsUtils.onButtonEvent;
+
+var isWebXRAvailable = require('../utils/').device.isWebXRAvailable;
+
+var GAMEPAD_ID_PREFIX = 'magic-leap';
+
+var MAGICLEAP_CONTROLLER_MODEL_GLB_URL = 'https://cdn.aframe.io/controllers/magicleap/control.glb';
+
+/**
+ * Button IDs:
+ * 0 - trigger (intensity value from 0.5 to 1)
+ * 1 - grip
+ * 2 - touchpad
+ * 3 - menu (dispatch but better for menu options)
+ *
+ * Axis:
+ * 0 - touchpad x axis
+ * 1 - touchpad y axis
+ */
+var INPUT_MAPPING_WEBVR = {
```

Comment:
Do you think we can remove WebVR references? My understanding is that there are no legacy WebVR browsers with magicleap support

Reply:
Yes, I missed this! Thanks for catching it!